### ISSUE TITLE
deleting a category with assigned articles does not return an error message.

### ISSUE THREAD

**Issue by phiw13** [textpattern/textpattern#1728]
See this forum post: https://forum.textpattern.com/viewtopic.php?pid=331405#p331405 for details

apparently no string exist for an eventual error message

**Comment by bloatware** [comment]
> apparently no string exist for an eventual error message

and that's the main obstacle.

**Comment by phiw13** [comment]
Sounds like an easy fix then for someone with access to insert the string, then.

4.8.8 polish stuff?

**Comment by petecooper** [comment]
@Bloke @bloatware - viable candidate for 4.8.8?

**Comment by Bloke** [comment]
Yep.

**Comment by phiw13** [comment]
This seems to work after your patch (https://github.com/textpattern/textpattern/commit/5417c88d3315ef2cd8e177f0d09bacd592ee0480). Attempting to delete a category with child categories or with articles assigned now returns a notification message flagging the failure.

One might quibble about the *type* of message. Currently I see a `warning` (amber). Trying to delete a section with articles on the other hand returns an `error`(red).

**Comment by bloatware** [comment]
Fair enough, [done](https://github.com/textpattern/textpattern/commit/febd9aaa756372895177bf0d029d3965065e6092), thanks.
